### ISSUE TITLE
Color contrast issues in 1.3.1

### ISSUE THREAD

**Issue by beyackle** [microsoft/BotFramework-Composer#6463]
## Describe the bug

We have a number of places that are failing color-contrast checks; they'd be easily handled as a group.

## Version

1.3.1

## Screenshots

Bot name and locale - this white text is failing contrast checks against the light blue, and also the light blue should be more distinctive against the slightly darker blue background.
![image](https://user-images.githubusercontent.com/61990921/111549386-5098aa00-8739-11eb-9b2d-756b057c8baf.png)

The green should be darker here.
![image](https://user-images.githubusercontent.com/61990921/111550059-99049780-873a-11eb-930d-965d800d20a1.png)

The words "default language" are too light for their size (they could also be larger for readability reasons).
![image](https://user-images.githubusercontent.com/61990921/111550152-c18c9180-873a-11eb-94ba-42e39642f349.png)

**Comment by beyackle** [comment]
![Screen Shot 2021-03-17 at 2 58 54 PM](https://user-images.githubusercontent.com/61990921/112055380-00905d80-8b14-11eb-919f-78a4627420c2.png)
I'll be working from this sample provided by Sangwoo to make this work.
